src/interface/storage.thrift

```diff
@@ -337,6 +347,22 @@ struct GetUUIDResp {
     2: common.VertexID id,
 }
 
+struct BlockingSignRequest {
+    1: common.GraphSpaceID          space_id,
+    2: common.PartitionID           part_id,
```

Comment:
For blocking request,  we don't need to specify the part id.  All we need to do on meta is to fanout the request to all storaged hosts.   And each storaged block its leader part one by one.